src/main/java/uk/nhs/hee/tis/trainee/forms/api/AdminLtftResource.java

```diff
@@ -77,7 +77,8 @@ public ResponseEntity<String> getAdminLtftCount(
   @GetMapping
   ResponseEntity<PagedModel<LtftAdminSummaryDto>> getLtftAdminSummaries(
       @RequestParam(value = "status", required = false) Set<LifecycleState> states,
-      @PageableDefault(size = Integer.MAX_VALUE, sort = "created", direction = Direction.ASC)
+      @PageableDefault(size = Integer.MAX_VALUE, sort = "content.change.startDate",
+          direction = Direction.ASC)
```

Comment:
I guess it would be good to confirm with the users, but I would expect a descending order sort to possibly be more useful here.

Reply:
Unless I've got my thinking backwards, this is closest start date first - which feels like what we'd want?

Reply:
Ah right you are. I was thinking of all the past ones appearing at the top, but I presume those would be filtered out.